corehq/apps/events/views.py

```diff
@@ -408,23 +410,95 @@ class AttendeesConfigView(JSONResponseMixin, BaseUserSettingsView, BaseEventView
     @allow_remote_invocation
     def get(self, request, *args, **kwargs):
         return self.json_response({
-            "mobile_worker_attendee_enabled": mobile_worker_attendees_enabled(self.domain)
+            'mobile_worker_attendee_enabled': mobile_worker_attendees_enabled(self.domain)
         })
 
+
+class ConvertMobileWorkerAttendeesView(BaseUserSettingsView, BaseEventView):
+    urlname = "convert_mobile_workers"
+
     @allow_remote_invocation
-    def post(self, request, *args, **kwargs):
-        json_data = json.loads(request.body)
-        attendees_enabled = json_data['mobile_worker_attendee_enabled']
+    def get(self, request, *args, **kwargs):
+        attendees_enabled = not mobile_worker_attendees_enabled(self.domain)
         toggle_mobile_worker_attendees(self.domain, attendees_enabled)
```

Comment:
Should this function call be put in the tasks themselves? I'm just thinking for the scenario where a task may fail to enable/disable mobile workers but then the setting has already been toggled. The toggle button on the attendee list view would then show the wrong status of this setting.

Reply:
Good point. Will update.